.github/workflows/build-test-deploy.yaml

```diff
@@ -113,6 +113,17 @@ jobs:
       - uses: actions/setup-go@v1
         with:
           go-version: "1.14"
+
+      - uses: sigstore/cosign-installer@main
+        with:
+          cosign-release: "v1.2.1"
```

Comment:
Do we care what version we use? No strong opinions, but I guess I'd rather have it use latest to know if there are upcoming issues.